### PR TITLE
Move storage-policies repository under team-docs.

### PR DESCRIPTION
**Pull request by lgruen**
After this has been merged, we can delete the https://github.com/populationgenomics/storage-policies repository.

Also fix some linter errors.

### CODE REVIEW

File: storage_policies/README.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+*TOB-WGS* or *RDNow*, with separate buckets and permission groups. Below,
+`$STACK` is a placeholder for the name of that effort, e.g. `$STACK` =
+`tob-wgs`.
+
+Currently, all buckets reside in the `australia-southeast1` GCP region. It's
+therefore essential that all computation happens in that region too, to avoid
+network egress costs.
+
+In general, all datasets within buckets should be versioned, using a simple
+major-minor naming scheme like `gs://cpg-$STACK-main/qc/v1.2/`. We don't have
```
Comment (pdiakumis) [inline storage_policies/README.md:42]: We started using https://github.com/iterative/dvc at UMCCR for data versioning, which may or may not interest you. Essentially it's like git for datasets (though I'll admit I haven't played with it too much).
Say you have v1.2 of a dir containing 10 files. And say in dir v1.3 you only update 1 of those files (e.g. add 5 rows), but the other 9 files are same between v1.2 and v1.3 i.e. you're duplicating data. Something like DVC would just keep track of the changed files (so would keep the 9 files, and the 2 versions of the single changed file).

File: storage_policies/README.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
+# Storage policies
+
+This document describes where our production datasets are stored, how [object
+lifecycles] are configured, and how access permissions are managed.
+
+[object lifecycles]: https://cloud.google.com/storage/docs/lifecycle
```
Comment (pdiakumis) [inline storage_policies/README.md:6]: Not sure if we want to stick to inline links or allow both ref and inline types?

File: storage_policies/README.md (hunk 1 above)
Comment (lgruen) [reply storage_policies/README.md:42]: Yes, DVC looks really cool and I've discussed this a bit with the Hail team :).

Unfortunately in our case with data in GCP buckets (way too large to store in git directly), it doesn't quite work (as far as I can tell).

File: storage_policies/README.md (hunk 2 above)
Comment (lgruen) [reply storage_policies/README.md]: Good catch, thanks! Fixed.

**Review by pdiakumis — APPROVED** [review]
Thanks so much! Sorry for delay, forgot to approve.
Re: the description of the eight different buckets, I'm thinking maybe we could summarise that in a table row instead of a paragraph each since they seem to have similar info categories. I'll see how it looks and might open a PR. 
And I'll need to look into Pulumi - thanks again ;-)

**Pull request merged**